item.rb

```diff
@@ -0,0 +1,36 @@
+class Item
+  attr_accessor :publish_date, :archived, :authors, :genre
+  attr_reader :id
+
+  def initialize(publish_date, archived: false, id: 0)
+    @id = id.zero? ? rand(1..1000) : id
+    @publish_date = publish_date
+    @archived = archived
+    @authors = []
+    @label = nil
+    @genre = []
+  end
+
+  def can_be_archived?
+    true if @publish_date > 10
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived? == true
+  end
+
+  def add_author(author)
+    @authors << author
+    author.items << self unless author.items.include?(self)
+  end
+
+  def to_h
+    {
+      id: @id,
+      publish_date: @publish_date.to_s,
+      archived: @archived
+    }
+  end
+
+  attr_writer :label
+end
```

Comment:
- [ ] hello team great work on your project and testing it properly, but I think you skipped testing this file, kindly add a test for this file :+1:

Reply:
Hi @VICTOR-CHUKWU, Thanks for highlighting the issues. We will fix them and submit the pull request for another evaluation.